### DISCUSSION TITLE
Can you change the "Format" properties in VideoCaptureDevice

### DISCUSSION THREAD

**Discussion by meganalo** [bonsai-rx/bonsai#1830] [Q&A]
Is there any reason why such limited options are offered under the Format tab in properties for VideoCaptureDevice? Does anyone know whether/ how this could be changed? We are running into an issue where the computer cannot record from two cameras at 100fps without crashing but there doesn't seem to be a way to lower the frame rate in Bonsai. We know the frames can go lower (ideally, we would like to record at 30fps) because we are able to bring it down to 30 in the software associated with the cameras, but for some reason Bonsai won't allow this. 

![pic1](https://github.com/bonsai-rx/bonsai/assets/128488461/436a6760-62c1-4d15-a9b6-3db1b08e7b9c)
![pic2](https://github.com/bonsai-rx/bonsai/assets/128488461/f610a3e1-9ad6-4e06-bb79-16fe19ce2487)

**Comment by bruno-f-cruz** [comment]
These settings are the ones provided by the manufacturer s drivers via directshow interface. For different cameras they will be different and bonsai has no control over them. You might have better luck using the OpenCv powered operator 'CameraCapture'. Otherwise you can downsample in bonsai by using the the reactive nodes. For instance Sample, SampleInterval, WithLatestFrom or Slice.

Hope it helps!

**Reply by meganalo** [reply]
Thank you! This was very helpful, I used SampleInterval to downsample and I was able to record and save from both streams. I would like to get CameraCapture working as an alternative as well, but we currently have an issue where the image is very zoomed in and I cannot figure out how to resolve this, I tried adjusting properties but changing Zoom did nothing and changing FrameWidth/ Height caused crashes. Not sure if anyone else has experienced this!

**Reply by bruno-f-cruz** [reply]
I would ask the manufacturer for examples on how to set these properties via OpenCV. Since the backend is identical, if you have an example working in, say Python, we should be able to do it from Bonsai as well.
